src/ui/server/helpers/is-populated-array/index.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * isPopulatedArray
+ * Check if an array is populated
+ * @param {Array}
+ * @returns {Boolean}
+ */
+const isPopulatedArray = (arr?: Array<any>): boolean => {
```

Comment:
Can we simplify it to 

``` const isPopulatedArray = (arr?: Array<any>): boolean => arr && arr.length;

Reply:
@abhi-markan the code snippet is empty but I guess you would suggest this?

```js
const isPopulatedArray = (arr?: Array<any>): boolean => arr && arr.length ?  true : false;
```

We could do this but I think it's less readable?

Reply:
Apologies, I was suggesting 

```
const isPopulatedArray = (arr?: Array<any>): boolean => arr && arr.length;

Reply:
That won't work though because it could return undefined whereas we need a boolean or number return type. Ttypecsript is flagging issues, tried a few things out, I think the original approach is the most readable - the tenary appraoch I posted would work but feel it's less readable.